<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditDelWindow</class>
 <widget class="QMainWindow" name="AddEditDelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">AddEditDelWindow {
    background-color: #2c3e50;
    border-radius: 10px;
    padding: 20px;
}
QLabel, QPushButton, QLineEdit, QComboBox, QSpinBox {
    color: white;
    font-size: 14px;
    margin-bottom: 10px;
}
QPushButton {
    background-color: #3498db;
    border: none;
    padding: 10px 20px;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #2980b9;
}
QLineEdit, QSpinBox {
    background-color: #34495e;
    border: none;
    padding: 5px;
    border-radius: 5px;
    selection-background-color: #3498db;
}
QSpinBox {
    width: 100px;  /* Increase the width */
    height: 40px;  /* Increase the height */
}
QLabel {
    font-weight: bold;
}
QComboBox {
    background-color: #34495e;
    border: none;
    padding: 10px;
    border-radius: 5px;
}
QTableWidget {
    background-color: #2c3e50;
    border-radius: 5px;
    gridline-color: #2980b9;
    color: white;
    selection-background-color: #2980b9;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="AddFile">
       <attribute name="title">
        <string>Ajouter fichier</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Type de fichier</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBoxFileType">
          <item>
           <property name="text">
            <string>Contigue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Chainee</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nom du fichier</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Add File Data</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="AddFileName"/>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="GenStudents">
          <property name="text">
           <string>Generate Students</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nombre des records</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ModFile">
       <attribute name="title">
        <string>Modifier Ficher</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="6" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Modify</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QTableWidget" name="tableWidget_2"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>163</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Generate modification Table</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>File First Block to modify</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete Record by Id</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
